backend/endpoints/user.py

```diff
@@ -94,7 +94,7 @@ def get_user(request: Request, id: int) -> UserSchema:
     return user
 
 
-@protected_route(router.put, "/users/{id}", ["users.write"])
+@protected_route(router.put, "/users/{id}")
```

Comment:
you still want to protect this endpoint so its only available to authed users

```suggestion
@protected_route(router.put, "/users/{id}", ["me.write"])
```